### DISCUSSION TITLE
Future cockpit parts

### DISCUSSION THREAD

**Discussion by Phantom088** [Drag0nD3str0yer/Moderately-Plane-Related#26] [Q&A]
What cockpit parts are planned to be added to this mod in the future?
Since the F-100 cockpit is there, do you have plans for other Century Series aircraft (such as the Lockheed F-104 Starfighter, McDonnell F-101A and B Voodoo, Convair F-102 Delta Dagger, Republic F-105 Thunderchief, and Convair F-106 Delta Dart)?

Have you also thought of some cockpits from Navy aircraft of that era, such as the F4D-1 Skyray, F5D-1 Skylancer, F-8 Crusader, F-3H Demon, F-9 Cougar, and FJ-4B Fury?

**Comment by Wolfosito** [comment]
F-106 and 102 would be great
An F-8 (A-7 perhaps) cockpit would be great in a Mk3 form factor too, the other A-7 cockpit out there (In Eskandare's mod iirc) didn't look good because it had to turn back to the round default fuselage quickly

**Reply by Phantom088** [reply]
There's a part of me that feels like a Mig-23, F-5E (with alternate F-20 cockpit that you can select), and M,ig-21 cockpit would also work REALLY well

**Reply by Wolfosito** [reply]
We're going in a bit heavy on the requests
I did make a MiG-23 cockpit model for a sideways mk3 shape though (Cold war aerospace mod in mind)
Hopefully I didn't mess up too much making it but here it is in case you guys wanna use it
[https://www.mediafire.com/file/f9f1s23abcfa9ae/mig23.blend/file](url)
of course a mig-23 cockpit wouldnt be complete without a 27, which I guess I could also make

**Reply by Phantom088** [reply]
Sorry about that
But yeah, the Mig-23/27 cockpit would work really well in-game

**Reply by yukikaze-zero** [reply]
I only want a 2 seater f-104 cockpit

**Reply by Drag0nD3str0yer** [reply]
> We're going in a bit heavy on the requests I did make a MiG-23 cockpit model for a sideways mk3 shape though (Cold war aerospace mod in mind) Hopefully I didn't mess up too much making it but here it is in case you guys wanna use it [https://www.mediafire.com/file/f9f1s23abcfa9ae/mig23.blend/file](url) of course a mig-23 cockpit wouldnt be complete without a 27, which I guess I could also make

actually no thats not too bad, ive got more than that on my to do list, ive just been too busy in recent to do much work

**Reply by yukikaze-zero** [reply]
Cool
